ansible/group_vars/nimbus.sepolia.yml

```diff
@@ -89,7 +99,6 @@ validator_client_keymanager_token: '{{lookup("bitwarden", "nimbus/keymanager", f
 validator_client_dist_validators_enabled: '{{ node.start is defined and node.end is defined }}'
 validator_client_dist_validators_start: '{{ (not node.vc) | ternary(0, node.start) | mandatory }}'
 validator_client_dist_validators_end:   '{{ (not node.vc) | ternary(0, node.end)   | mandatory }}'
-
```

Comment:
Why was this empty line separating different configuration removed?

Reply:
my bad, added this line break back